Hydra4NET/QueueProcessor.cs

```diff
@@ -53,8 +52,9 @@ public void Init(CancellationToken ct = default)
                 {
                     CalculateSlidingDuration();
                 }
-                _timer.Change((int)_slidingDuration, 0);
+                _timer?.Change((int)_slidingDuration, 0);
```

Comment:
while this is correct, we can also do timer!.Change to assert that is not null. Since the timer is initialized when this code is running, this approach may be more appropriate